### DISCUSSION TITLE
Is there any way to auto release the collection/partition after a configured "time-to-live" period?

### DISCUSSION THREAD

**Discussion by xiaobingxia-at** [milvus-io/milvus#37488] [Q&A and General discussion]
As title described, Is there any way to auto release the collection/partition after a configured "time-to-live" period?.

**Comment by xiaofan-luan** [comment]
No. time to live will delete the data after compaction happened.

On opensource you can config LRU on each segment (but this not stable and not released yet).
On Zilliz cloud we offer tiered storage which can put hot and cold data into different storage.

**Reply by xiaobingxia-at** [reply]
Thank you! Because I have a huge amount of data , but only minor percentage of them is alive. I wish I could load them, after a few days, if no one use them, just release them. I'm thinking:
1. If we do client.load_partitions(...), we can have a TTL config, that it could auto-release from cache if no one use them
2. OR, we can build a cron job to poll Milvus daily, for each partition, to get when is the last time it got accessed. And release them if needed.
Do you think we can do either of above 2 ways to solve this problem?
